src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const smallImg = document.querySelectorAll('.list-item__link');
+
+const imagesLinksArray = Array.from(smallImg).map(
+  (item) => item.getAttribute('href'),
+  {},
+);
+
+smallImg.forEach(
+  (item, i) =>
+    item.addEventListener('click', function (e) {
+      e.preventDefault();
+
+      largeImg.setAttribute('src', `${imagesLinksArray[i]}`);
+    }),
+  {},
+);
```

Comment:
Similarly to the previous comment, the second argument to `forEach` should be removed as it is not needed and does not serve any purpose.